src/Components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Button extends Component {
+  render() {
+    return (
+      <div>{ this.props.label }</div>
+    );
+  }
+}
+
+Button.propTypes = { label: PropTypes.string };
```

Comment:
Let's make this a functional component, please.
This should be recieving the name prop from button pannel.